[2022-09-05 19:57:22,496] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: my_dag_postgres2.insert_into_table manual__2022-09-05T19:56:22.560677+00:00 [queued]>
[2022-09-05 19:57:22,511] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: my_dag_postgres2.insert_into_table manual__2022-09-05T19:56:22.560677+00:00 [queued]>
[2022-09-05 19:57:22,512] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-09-05 19:57:22,513] {taskinstance.py:1377} INFO - Starting attempt 2 of 2
[2022-09-05 19:57:22,514] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-09-05 19:57:22,534] {taskinstance.py:1397} INFO - Executing <Task(PostgresOperator): insert_into_table> on 2022-09-05 19:56:22.560677+00:00
[2022-09-05 19:57:22,541] {standard_task_runner.py:52} INFO - Started process 2306 to run task
[2022-09-05 19:57:22,546] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'my_dag_postgres2', 'insert_into_table', 'manual__2022-09-05T19:56:22.560677+00:00', '--job-id', '3825', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_postgres.py', '--cfg-path', '/tmp/tmp7jsz1ki1', '--error-file', '/tmp/tmpryt_3cio']
[2022-09-05 19:57:22,547] {standard_task_runner.py:80} INFO - Job 3825: Subtask insert_into_table
[2022-09-05 19:57:22,634] {task_command.py:371} INFO - Running <TaskInstance: my_dag_postgres2.insert_into_table manual__2022-09-05T19:56:22.560677+00:00 [running]> on host eba23da77ca5
[2022-09-05 19:57:22,730] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=my_dag_postgres2
AIRFLOW_CTX_TASK_ID=insert_into_table
AIRFLOW_CTX_EXECUTION_DATE=2022-09-05T19:56:22.560677+00:00
AIRFLOW_CTX_TRY_NUMBER=2
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-09-05T19:56:22.560677+00:00
[2022-09-05 19:57:22,747] {base.py:68} INFO - Using connection ID 'postgres_localhost' for task execution.
[2022-09-05 19:57:22,759] {dbapi.py:231} INFO - Running statement: 
            insert into dag_runs (dt, dag_id) values ('2022-09-05', 'my_dag_postgres2')
            , parameters: None
[2022-09-05 19:57:22,764] {dbapi.py:239} INFO - Rows affected: 1
[2022-09-05 19:57:22,792] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=my_dag_postgres2, task_id=insert_into_table, execution_date=20220905T195622, start_date=20220905T195722, end_date=20220905T195722
[2022-09-05 19:57:22,839] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-09-05 19:57:22,882] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
